oxidation/bindings/generator/templates/binding_electron_meths.rs.j2

```diff
@@ -466,6 +478,12 @@ fn {{ meth.name }}(mut cx: FunctionContext) -> JsResult<JsPromise> {
 
 #[allow(unused_imports)]
 use neon::{prelude::*, types::buffer::TypedArray};
+#[allow(dead_code)]
+const I32_MAX: f64 = i32::MAX as f64;
+#[allow(dead_code)]
+const I32_MIN: f64 = i32::MIN as f64;
+const U32_MAX: f64 = u32::MAX as f64;
```

Comment:
no need for `#[allow(dead_code)]` for U32_* ?

Don't we want to just inline `i32::MIN as f64` in the callsite to not have to wonder about that ?

Reply:
`U32_* is already used`